cli/cli_tests.sh

```diff
@@ -186,7 +185,7 @@ echo "Running data submission step"
 echo "====================================="
 medperf dataset submit -p $PREP_UID -d $DIRECTORY/dataset_a -l $DIRECTORY/dataset_a --name="dataset_a" --description="mock dataset a" --location="mock location a" -y
 checkFailed "Data submission step failed"
-DSET_A_UID=$(medperf dataset ls | grep dataset_a | tr -s ' ' | cut -d ' ' -f 1)
+DSET_A_UID=$(medperf dataset ls | grep dataset_a | tr -s ' ' | awk '{$1=$1;print}' | cut -d ' ' -f 1)
```

Comment:
Looks good, but why the previous version failed?

Reply:
previously, unregistered entities were also listed by default (so this will list the demo dataset hash), but currently no (since we now require --unregistered to list unregistered only). unregistered entities have a long hash instead of ID.
due to how we use python's `tabulate`, having a long entry in one of the rows will remove leading spaces of other shorter entries, hence we didn't want the `awk` command to remove leading spaces. but now all entries are short so you may have leading spaces